packages/core/src/modules/credentials/services/CredentialService.ts

```diff
@@ -672,6 +672,20 @@ export class CredentialService {
       credentialDefinition,
       revocationRegistryDefinition,
     })
+
+    // If we have the rev_reg_id, then we also want to set the cred_rev_id, which only can be fetched by retrieving credential from indy
+    if (indyCredential.rev_reg_id) {
```

Comment:
It seems you're re-fetching the credential here? I'm not sure what you mean with the comment on re-fetching in that case

Reply:
The point of the comment was to communicate why we were refetching the credential when we had just received it. It appears that the comment didn't quite communicate that 😆